README.md

```diff
@@ -19,10 +19,11 @@ download the binary for your platform.
 
 ## Usage
 
-Snag works by reading a `.snag.yml` file which contains a set of
-commands, ignored directories, and options.
+Snag works by reading a yaml file name `.snag.yml`. It allows you to configure what snag will
```

Comment:
Do you mean `Snag works by reading a named .snag.yml`? That sounds a little funky